### ISSUE TITLE
Stacks are not accounted for

### ISSUE THREAD

**Issue by Naxesss** [Naxesss/MapsetParser#1]
An object being within a certain radius of another will move that other object up and left slightly. This behaves differently for things like slider tails, and only applies between circles and sliders (possibly also spinners if you put an object right in the center, unsure). When the object stacked on fades out, the stack should stay. This not only affects the object visually but also in gameplay, moving its hitbox.

So basically I'll need to find
- The radius
- How this works for slider tails
- If it applies for spinners too
- How far up to the left the object moves
- How that behaves with varying circle size
